style.css

```diff
@@ -0,0 +1,34 @@
+@import url('https://fonts.googleapis.com/css2?family=Asap:wght@600&family=Lato:wght@300;400;700&family=Lobster&family=Playfair+Display&display=swap');
+@import url(css/homepage.css);
+
+:root{
+    /* Font family */
+    --brand-font: 'Lobster', cursive;
+    --primary-font: 'Playfair Display', serif; 
+    --secondary-font: 'Asap', sans-serif; 
+    --ternary-font: 'Lato', sans-serif;  
+    /* 
+    font-family: 'Lobster', cursive;            B
+    font-family: 'Playfair Display', serif;     Deal of the day
+    font-family: 'Asap', sans-serif;            BRAND/PRICE/COLOR
+    font-family: 'Lato', sans-serif;            Paragraph 300/400/700
+    */
```

Comment:
remove unnecessary blocks of code before merging

Reply:
Actually, it is not unnecessary. I have kept it to remember when to use which font. Is it ok if I keep it?

Reply:
it might create misunderstandings in middle of the code
if you wish to keep it, do place it at the bottom/top-most